### PR TITLE
Added new Controller and required properties.

### PR DESCRIPTION
**Pull request by CH-JonathanBlake**
__Short description outlining key changes/additions__
Added new controller to redirect users attempting to access the old home page.
__JIRA Ticket Number__
https://companieshouse.atlassian.net/browse/FA-1035
### Type of change

* [ ] Bug fix
* [X] New feature
* [ ] Breaking change

### Pull request checklist

* [X] I have added unit tests for new code that I have added
* [X] I have added/updated functional tests where appropriate
* [X] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Review by ICraig-CH — APPROVED** [review]

**Review by ch-rfishwick — APPROVED** [review]

**Comment by ch-rfishwick** [comment]
Didn't notice the conflicts when I approved. Happy to review again after the conflicts have been resolved if necessary although if it's just the pom it might be something minor

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -6,6 +6,8 @@ piwik.siteId=${PIWIK_SITE_ID}
 enquiries=mailto:enquiries@companieshouse.gov.uk
 home.path=dev-hub/home
 home.url=
+absoluteHome.url=${REDIRECT_URI}
+historicHome.url=api/docs
```
Comment (Zain-Abbas1) [inline src/main/resources/application.properties:10]: Should this have a preceding slash `/api...`?

Comment (CH-JonathanBlake) [reply src/main/resources/application.properties:10]: Doesn't seem to need it but it would be more consistent.

**Comment by CH-JonathanBlake** [comment]
> Didn't notice the conflicts when I approved. Happy to review again after the conflicts have been resolved if necessary although if it's just the pom it might be something minor

It got confused by the reordering of elements in the pom I think. I just overwrote it all with the Master POM and re-added the field.

**Review by ICraig-CH — APPROVED** [review]

**Review by ch-rfishwick — APPROVED** [review]

**Pull request merged**